aiohttp/client_reqrep.py

```diff
@@ -1144,6 +1150,7 @@ async def json(
         return loads(self._body.decode(encoding))  # type: ignore[union-attr]
 
     async def __aenter__(self) -> "ClientResponse":
+        self._in_context = True
```

Comment:
Is there any way they could enter/exit multiple time?

Reply:
Once exited, the response is already released. So, doesn't really matter after that point whether we call .release() or not.